specification/automation/resource-manager/Microsoft.Automation/preview/2023-05-15-preview/examples/job/getJobStream.json

```diff
@@ -0,0 +1,26 @@
+{
+  "parameters": {
+    "subscriptionId": "51766542-3ed7-4a72-a187-0c8ab644ddab",
+    "resourceGroupName": "mygroup",
+    "automationAccountName": "ContoseAutomationAccount",
+    "jobName": "foo",
+    "jobStreamId": "851b2101-686f-40e2-8a4b-5b8df08afbd1_00636535684910693884_00000000000000000001",
+    "api-version": "2023-05-15-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "id": "/subscriptions/51766542-3ed7-4a72-a187-0c8ab644ddab/resourcegroups/mygroup/providers/Microsoft.Automation/automationAccounts/ContoseAutomationAccount/jobs/jobName/streams/851b2101-686f-40e2-8a4b-5b8df08afbd1_00636535684910693884_00000000000000000001",
+        "properties": {
+          "jobStreamId": "851b2101-686f-40e2-8a4b-5b8df08afbd1:00636535684910693884:00000000000000000001",
+          "summary": null,
```

Comment:
>"summary": null, [](http://example.com/codeflow?start=10&length=16)

avoid string nullable properties

>Set "x-nullable": true on a schema to indicate that a null is a legal value. By default, a null value should be disallowed when forming a request and rejected during payload deserialization.

For arrays, sending/receiving a null array entry is not supported and should result in an error.

Reply:
Marked "x-nullable": true for summary